selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/utils/BrowserLogsUtil.java

```diff
@@ -25,27 +27,115 @@
  * Read and store browser logs to the test logs. Log level and type are defined in {@link
  * org.eclipse.che.selenium.core.SeleniumWebDriver#doCreateDriver(URL)}
  */
-@Singleton
 public class BrowserLogsUtil {
-  private final SeleniumWebDriver seleniumWebDriver;
-  private final Logger LOG = LoggerFactory.getLogger(BrowserLogsUtil.class);
-
-  @Inject
-  public BrowserLogsUtil(SeleniumWebDriver seleniumWebDriver) {
```

Comment:
We are injecting `seleniumWebDriver` here avoid injection it into the test class.